README.md

```diff
@@ -1,25 +1,27 @@
 # daskhub-rhg-config
-Daskhub-flavored Jupyterhub deployment configuration.
+Daskhub-flavored Jupyterhub deployment configuration for argocd.
 
 This is a demonstration sandbox.
 
-
+## Fresh deployment
```

Comment:
Will this production "fresh deployment" snippet need to be invoked manually after each tagged version? Or will we have a different argocd command than "app create" which says "modify the existing app to point to a new tag". Might be useful to put that info in the readme

Reply:
Good idea! I added a bit on that in `26a61f94abb8c8c8461ed0e1e964bc4b8bd44435`.